web/rootfs/usr/bin/init_jitsi_web

```diff
@@ -25,8 +21,9 @@ if [[ $DISABLE_HTTPS -ne 1 ]]; then
             $STAGING \
             --issue \
             --standalone \
-            --pre-hook "if [[ -d /var/run/s6/services/nginx ]]; then s6-svc -d /var/run/s6/services/nginx; fi" \
-            --post-hook "if [[ -d /var/run/s6/services/nginx ]]; then s6-svc -u /var/run/s6/services/nginx; fi" \
+            # TODO: replace s6 service logic with supervisord
```

Comment:
This still valid?

Reply:
It is now that I fixed my bash syntax :)

Example stopping `saslauthd` on `prosody` container:

``` bash
$ if ! supervisorctl pid saslauthd ; then supervisorctl start saslauthd; fi
15404
I have no name!@shard-0-prosody-775d5b55d8-bzbcn:/$ supervisorctl status
prosody                          RUNNING   pid 53, uptime 4 days, 2:02:49
saslauthd                        RUNNING   pid 15404, uptime 0:00:19
```

Reply:
```suggestion
```